pkg/agent/config.go

```diff
@@ -130,15 +127,6 @@ func (c *Config) Dump() (string, error) {
 func (c *Config) validate() error {
 	var result *multierror.Error
 
-	if c.Token == "" {
-		if c.OrganizationID == "" {
-			result = multierror.Append(result, fmt.Errorf("organization_id is required"))
-		}
-		if c.ClusterID == "" {
-			result = multierror.Append(result, fmt.Errorf("cluster_id is required"))
-		}
-	}
-
```

Comment:
The checks inside the `if` should be applied all the times now.

Reply:
Oh Right! I messed up there, I'll add the conditions back